starter_scripts/exaslct_within_docker_container.sh

```diff
@@ -112,7 +112,7 @@ function _get_mount_point_paths() {
 }
 
 declare -a mount_point_paths
-_get_mount_point_paths "${@}"
+_get_mount_point_paths ${@//=/ }
```

Comment:
What happens, if you have '=' as a delimiter and in the value? For example, ./exaslct --flavor-path=flavors/test=test

Reply:
We also need a test for this.